packages/ui/src/components/Visualization/ContextToolbar/FlowType/NewFlow.tsx

```diff
@@ -48,10 +48,10 @@ export const NewFlow: FunctionComponent<PropsWithChildren> = () => {
         <span className="pf-v5-u-m-sm">New</span>
       </FlowTypeSelector>
       <Modal
+        variant={ModalVariant.large}

```

Comment:
@kumaradityaraj , since this is just a small `Continue / Cancel` question, better to stay with small 

```suggestion
        variant={ModalVariant.small}
```